Src/Support/GoogleApis/Apis/[Media]/Download/MediaDownloader.cs

```diff
@@ -166,96 +175,86 @@ public IDownloadProgress Download(string url, Stream stream)
                 throw new ArgumentException("stream doesn't support write operations");
             }
 
-            RequestBuilder builder = null;
-
-            var uri = new Uri(url);
-            if (string.IsNullOrEmpty(uri.Query))
+            // Add alt=media to the query parameters.
+            var uri = new UriBuilder(url);
+            if (uri.Query == null || uri.Query.Length <= 1)

```

Comment:
The change to this condition suggests that we sometimes see a single-character query here, which we want to replace. What's the background to this?